x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/apikey/QueryApiKeyRequest.java

```diff
@@ -62,24 +60,6 @@ public QueryApiKeyRequest(
         this.withLimitedBy = withLimitedBy;
     }
 
-    public QueryApiKeyRequest(StreamInput in) throws IOException {
-        super(in);
-        this.queryBuilder = in.readOptionalNamedWriteable(QueryBuilder.class);
-        this.from = in.readOptionalVInt();
-        this.size = in.readOptionalVInt();
-        if (in.readBoolean()) {
-            this.fieldSortBuilders = in.readCollectionAsList(FieldSortBuilder::new);
-        } else {
-            this.fieldSortBuilders = null;
-        }
-        this.searchAfterBuilder = in.readOptionalWriteable(SearchAfterBuilder::new);
-        if (in.getTransportVersion().onOrAfter(TransportVersions.V_8_5_0)) {
-            this.withLimitedBy = in.readBoolean();
-        } else {
-            this.withLimitedBy = false;
-        }
-    }
-
```

Comment:
This is an unrelated sneak. This is not used anymore. This should've been removed in https://github.com/elastic/elasticsearch/pull/104653 .